### ISSUE TITLE
VLC is unable to open the MRL: rtsp.....

### ISSUE THREAD

**Issue by WillyBilly90** [kakone/VLC.MediaElement#62]
I tried to use the VLC.MediaElement to view my foscam camera in a UWP app, but I get the error VLC is unable to open the MRL: RTSP with the username, password and ip address of the camera.

The RTSP stream works fine in the VLC mediaplayer desktop app, so that should not be the problem.

I added the VLC logfile of my UWP app, maybe somebody can tell me what I do wrong.
In the logfile I replaced the username and password for username:password for safety.

[export.txt](https://github.com/kakone/VLC.MediaElement/files/1908213/export.txt)

**Comment by kakone** [comment]
Yes, there was a bug in the current version (cf [vlc-winrt #238](https://code.videolan.org/videolan/vlc-winrt/issues/238)). I'm going to do a new version that should fix this problem.

**Comment by WillyBilly90** [comment]
When can we expect the new version?

**Comment by kakone** [comment]
I wanted to do it today but I have a problem compiling VLC for WinRT. I will ask the VLC team...

**Comment by WillyBilly90** [comment]
Did you allready received an answer of the VLC team ?

**Comment by Upd4ting** [comment]
Do you know when this bug will be fixed? I'm currently coding in a Hackaton and I really need that working

**Comment by kakone** [comment]
Sorry for the delay, I was very busy.
I did a [new version](https://www.nuget.org/packages/VLC.MediaElement/3.0.1.2) (I let you test it). I updated the x86 and x64 with [latest nightly builds](http://nightlies.videolan.org/build/winrt-x86_64/).

@CliffAgius It's not easy to reference a build of the libVLC because of the differents libraries for each architecture. But, you can try to build your own libvlcx nuget package with the source code [here](https://github.com/kakone/libVLCX).

Next time, I hope that I should be able to use the official [libvlc nuget package for WinRT](https://github.com/mfkl/libvlc-nuget) (not available for the moment).

**Comment by WillyBilly90** [comment]
Does it work for ARM too ? I would like to use it on a Raspberry Pi

**Comment by claudiosanchez** [comment]
I am encountering the issue when using the library with Windows 10 IoT Core running on a Raspberry Pi 3 B. 
Nuget package:VLC.MediaElement 3.0.1.2
